pikaur/print_department.py

```diff
@@ -448,7 +448,10 @@ def _bold_line(line):
             new_aur_deps,
             verbose=verbose, color=color, print_repo=False
         ))
-    result += ['']
+    if PikaurConfig().sync.ShowDownloadSize.get_bool():
+        result += _get_total_size(install_info)
```

Comment:
and formatting happen here, since formatting is not related to InstallinfoFetcher

Reply:
updated